src/olympia/amo/fields.py

```diff
@@ -38,7 +39,21 @@ def __call__(self, value):
 
 
 class HttpHttpsOnlyURLField(fields.URLField):
-    default_validators = [URLValidatorBackport(schemes=('http', 'https'))]
+
+    def __init__(self, *args, **kwargs):
+        super(HttpHttpsOnlyURLField, self).__init__(*args, **kwargs)
+
+        self.validators = [
```

Comment:
The only way to use `override_settings` is to do that, otherwise `settings` gets defined before everything (I suppose because we use a class attribute).